assets/js/state/selectors/cluster.js

```diff
@@ -24,6 +24,11 @@ export const getClusterName = (clusterID) => (state) => {
   return get(cluster, 'name', '');
 };
 
+export const getClusterByHost = (state, hostID) => {
```

Comment:
Moved from `index`. Now it uses lodash find function

Reply:
YEEEEEEEAH